src/pages/administration/students/index.vue

```diff
@@ -262,6 +263,13 @@ export default {
 		schoolInternallyManaged() {
 			return !this.school?.ldapSchoolIdentifier && !this.school?.source;
 		},
+		getTableActions() {
```

Comment:
```suggestion
		getTableActions() {
```
maybe "get" in the computed name is a bit misleading. I would assume that get() is a function. All the other computed props we have do not use this schema. 
Maybe rename it to something like `permissionFilteredTableActions` (or something shorter :D)